presets/react/storybook/preview-ssr.js

```diff
@@ -0,0 +1,3 @@
+import setup from './preview';
```

Comment:
what is the purpose of this file?

Reply:
storybook loads preview.js when running the preview in the browser. it is used for preview configuration and initialisation (you could initialize stories here too). I splitted preview.js from preview-ssr.js because there are some limitations in ssr preview. 
For example: You cannot load additional stories. We would need them on server side too.

That's why I want subpackages to have specific configs for preview and ssr preview. Would lead to errors that are hard to debug if you don't know all internals of storybook